test/unit/HashConsensus.t.sol

```diff
@@ -530,4 +546,77 @@ contract HashConsensusInternalUnitTest is HashConsensusUnitTest {
         emit HashConsensusWrapper.OnlyMemberModifierPassed();
         _hashConsensusWrapper.onlyMemberProtected();
     }
+
+    function test_updateQuorumRevertsIfHashIsUsed() public {
```

Comment:
Use underscores to separate the method name and condition for better readability. Also, adopt the `RevertOn` convention for tests that check revert behavior: `test_updateQuorumReverts_RevertOn_HashIsUsed()`.